frontend/package.json

```diff
@@ -44,7 +44,16 @@
   },
```

Comment:
Please add a command to run linter in this file

Reply:
@zgr92 added